docs/api/sql/Raster-visualizer.md

```diff
@@ -20,7 +22,31 @@ Output:
 iVBORw0KGgoAAAA...
 ```
 
-## Visualize a Raster band as a 2D matrix
+### RS_AsImage
+Introduction: Returns a HTML that when rendered using an HTML viewer or via a Jupyter Notebook, displays the raster as a square image of side length `imageWidth`. Optionally, an imageWidth parameter can be passed to RS_AsImage in order to increase the size of the rendered image (default: 200).
+
+Since: `1.5.0`
+
+Format: `RS_AsImage(raster: Raster, imageWidth: int = 200)`
+
+Spark SQL Example:
+
+```sql
+SELECT RS_AsImage(raster, 500) from rasters
+SELECT RS_AsImage(raster) from rasters
+```
+
+!!!Tip
```

Comment:
This needs to be updated to SedonaUtils. Please give a short example about how to use SedonaUtils.display_image(df)